nengo/tests/test_processes.py

```diff
@@ -99,7 +100,7 @@ def test_gaussian_whitenoise(Simulator, rms, seed, plt):
 def test_whitesignal_rms(Simulator, rms, seed, plt):
     t = 1.
     d = 500
-    process = WhiteSignal(t, rms=rms)
+    process = WhiteSignal(t, high=500, rms=rms)
```

Comment:
It's [PEP-3102](https://www.python.org/dev/peps/pep-3102/)... googling around a bit, I think it's in all of Python 3? So I misspoke in saying that it's 3.5 specific!